Examples/NewModuleTemplate/plasterManifest.xml

```diff
@@ -53,6 +53,10 @@
         </parameter>
     </parameters>
     <content>
+        <message>
+Scaffold a PowerShell Module with the files required to run Pester tests, build with PSake and publish to the PSGallery.
+
```

Comment:
That extra line is because I wanted a blank line between this message and the output that Invoke-Plaster generates.  That said, the ending (and beginning) newline is an interesting issue.  We do write the message element content with a final newline because underneath, we just throw the string at `Write-Host` which will automatically append a newline.  In fact, I will probably add a `nonewline` attribute to the `message` element so you can conditionally construct a single line with multiple message elements.  

That sounds pretty straight forward.  Where it gets a little dodgy is this code I added which I'm still on the fence about:
```powershell
# Eliminate whitespace before and after the text that just happens to get inserted because you want
# the text on different lines than the start/end element tags.
$trimmedText = $text -replace '^[ \t]*\n','' -replace '\n[ \t]*$',''
```
I was getting bit by specifying my XML like this:
```xml
    <message>
Long message string here
    </message>
```
This would wind up with a newline before the string along with any whitespace that is after `<message>` but before the newline e.g. `<message>____ `.  Same for whitespace before the closing tag `____</message>`.  

The `replace` operators are used to remove just the whitespace up to the "first" newline after `<message>` and the last newline and whitespace after it but before `</message>`.

It strikes me that perhaps I'm trying to be too clever and perhaps should just obey the KISS principle.